gslb/ingestion/event_handlers.go

```diff
@@ -122,13 +129,19 @@ func AddLBSvcEventHandler(numWorkers uint32, c *GSLBMemberController) cache.Reso
 						fetchedSvc.Name, gslbutils.ObjectDelete, fetchedSvc.Hostname, fetchedSvc.Tenant, c.workqueue)
 					return
 				}
-				oldSvcMeta, ok := k8sobjects.GetSvcMeta(oldSvc, c.name)
-				// check if tenant has changed for service
-				if ok && oldSvcMeta.Tenant != svcMeta.Tenant {
-					oper := gslbutils.ObjectDelete
-					publishKeyToGraphLayer(numWorkers, gslbutils.SvcType, c.name, oldSvcMeta.Namespace, oldSvcMeta.Name,
-						oper, oldSvcMeta.Hostname, oldSvcMeta.Tenant, c.workqueue)
+				if !gslbutils.CheckTenant(svc.Namespace, c.name, svcMeta.Tenant) {
```

Comment:
I think we should delete older objects only once we have the updated acceptable SVC/ing/route  mainly to avoid delete operation before SVC/Ingress update is completed by AKO

Reply:
Can we please document this behaviour?